spec/requests/article_spec.rb

```diff
@@ -0,0 +1,196 @@
+require 'rails_helper'
+
+RSpec.describe "Articles", type: :request do
+  let(:initial_attributes) do
+    { title: 'test', body: 'test body'}
+  end
+
+  let(:valid_attributes) do
+    { title: 'new test', body: 'new body' }
+  end
+
+  let(:nil_title_attributes) do
+    { title: nil, body: 'new body' }
+  end
+
+  let(:nil_body_attributes) do
+    { title: 'new title', body: nil }
+  end
+
+  describe "GET :index" do
+    before(:each) do 
+      get articles_path
+    end
+
+    it 'returns with 200 success status code' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders the :index view' do
+      expect(response).to render_template(:index)
+    end
+  end
+
+  describe "GET :new" do
+    before(:each) do
+      get new_article_path
+    end
+
+    it 'returns with 200 success status code' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders the :new view' do
+      expect(response).to render_template(:new)
+    end
+  end
+
+  describe "POST :create" do
+    context 'with valid parameters' do
+      valid_article = { title: 'test1', body: 'test' }
+
+      it 'returns with a 302 found status code' do
+        post articles_path, params: { article: initial_attributes }
+
+        expect(response).to have_http_status(:found)
+      end
```

Comment:
Redundant test? Observe line 69: if an article can be redirected to, it must have a `found 302` response